.github/workflows/kind.yaml

```diff
@@ -50,6 +50,7 @@ jobs:
       - name: Enable Relay
         run: |
           cilium hubble enable --ui
+          cilium status --wait --wait-duration=5m
```

Comment:
Still needed since we now have `cilium status --wait` in next step?

Reply:
It wasn't there xD. Dropping the commit.